kotlin-jvm/src/main/kotlin/playground/Mordant.kt

```diff
@@ -0,0 +1,44 @@
+@file:Suppress("PackageDirectoryMismatch")
+
+package playground.mordant
+
+import com.github.ajalt.mordant.TermColors
+
+fun main() {
```

Comment:
Please add documentation like this, the rest of the PR looks good!

```
/**
 * ajalt/mordant -   Full-featured text styling for Kotlin command-line applications 

- [GitHub](https://github.com/ajalt/mordant)
 */
fun main() {
    println()
    println("ajalt/mordant -   Full-featured text styling for Kotlin command-line applications")
```

Reply:
Yes, sure. will add those lines.

Reply:
@jmfayard added the documentation !